schema/bom-1.6.schema.json

```diff
@@ -661,34 +661,35 @@
             {
               "type": "array",
               "title": "Creation Tools (legacy)",
-              "description": "[Deprecated] The tool(s) used in the creation of the BOM.",
+              "description": "[Deprecated] The tool(s) used in the creation of the CycloneDX document.",
               "items": {"$ref": "#/definitions/tool"}
             }
           ]
         },
         "authors" :{
           "type": "array",
           "title": "Authors",
-          "description": "The person(s) who created the BOM. Authors are common in BOMs created through manual processes. BOMs created through automated means may not have authors.",
+          "description": "The person(s) who created the CycloneDX document.\nAuthors are common in documents created through manual processes. Documents created through automated means may not have authors.\nThis may be different from the author(s) of the component that the the CycloneDX document describes.",
           "items": {"$ref": "#/definitions/organizationalContact"}
         },
         "component": {
           "title": "Component",
-          "description": "The component that the BOM describes.",
+          "description": "The component that the the CycloneDX document describes.",
           "$ref": "#/definitions/component"
         },
         "manufacture": {
```

Comment:
I know this isn't part of your PR but I stumbled across it: But...`manufacture`, shouldn't this be `manufacturer`?

Reply:
In that case: While we're at it. Could you just add a comment here as well letting people know about this so we don't get repeated questions?

Reply:
done via 771cfe513f7c413c159423e875cb84ce2dd03a9a